input/sources/tev-000049.yaml

```diff
@@ -0,0 +1,10 @@
+source_id: 49
+
+tevcat_id: 75
+tevcat2_id: cIB7Uf
+tevcat_name: TeV J1104+382
+
+tgevcat_id: 49
+tgevcat_name: TeV J1104+3811
+
+papers:
```

Comment:
For the papers, I think it would be best to keep the `- none` from the example file.
It'll make it easier to add to the list of papers later (every source will have at least one paper)